Automattic/jetpack
# Packages: Introduce a jetpack-error package

This PR introduces a simple `jetpack-error` package.

Next step would be to replace any `Jetpack_Error` instances with `Automattic\Jetpack\Error`.

#### Changes proposed in this Pull Request:
* Introduce a jetpack-error package

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Part of the Jetpack DNA project - p1HpG7-70O-p2

#### Testing instructions:
* Checkout this branch.
* Navigate to the package directory: `cd packages/error`
* Run the package installation and tests: `composer phpunit`
* Verify tests pass.

#### Proposed changelog entry for your changes:
* Introduce a jetpack-error package
